cgi-bin/DW/Controller/Admin.pm

```diff
@@ -45,22 +45,6 @@ DW::Controller::Admin->_register_admin_pages_legacy(
     '/',
     [ 'faq/', '.admin.faq.link', '.admin.faq.text', [ 'faqadd', 'faqedit', 'faqcat' ] ],
     [ 'fileedit/', '.admin.file_edit.link', '.admin.file_edit.text', ['fileedit'] ],
-    [
-        'memcache', '.admin.memcache.link',
-        '.admin.memcache.text', [ 'siteadmin:memcacheview', 'siteadmin:*' ]
-    ],
-    [
-        'memcache_view',
-        '.admin.memcache_view.link',
-        '.admin.memcache_view.text',
-        [
-            'siteadmin:memcacheview',
-            'siteadmin:*',
-            sub {
-                return ( $LJ::IS_DEV_SERVER, LJ::Lang::ml("/admin/index.tt.devserver") );
-            }
-        ]
-    ],
     [
         'pay/',
```

Comment:
I realized when I saw the failed build status that tidyall wants to reformat the surrounding lines after removing this block, but that will definitely interfere with merging the admin/pay changes...

Reply:
It's probably OK to leave broken a few days while we merge... :s